tests/integration/awslambda/test_lambda_api.snapshot.json

```diff
@@ -8777,5 +8777,14 @@
         }
       }
     }
+  },
+  "tests/integration/awslambda/test_lambda_api.py::TestLambdaAccountSettings::test_account_settings_total_code_size": {
+    "recorded-date": "05-12-2022, 14:06:49",
+    "recorded-content": {
+      "total_code_size_diff_create_function": 264,
```

Comment:
We can snapshot it, but this is probably going to be flaky, since the creation of the zip archive is not reproducible, maybe not even in size. (different compression options, depending on the OS for example).
Would be nice to really only check if the size increases by a certain amount for example, not the exact value.
This is mostly relevant for zip files not checked-in to the repository, for example for the layer it is not relevant.

We might need to add a snapshot ignore for this values.

Reply:
right, I see that [create_zip_file](https://github.com/localstack/localstack/blob/75faee8adaeb07f6e5c76760e774889f66ad5ab9/localstack/utils/testutil.py#L164) has a comment about different zip sizes.

I did both for parity testing:

* `test_account_settings_total_code_size` now uses checked-in zip files with deterministic size
* `test_account_settings_total_code_size_config_update` checks for a rough size increase